### ISSUE TITLE
Deprecation warning in Rails 5

### ISSUE THREAD

**Issue by yurijmi** [adzap/validates_timeliness#143]
In [this file](https://github.com/adzap/validates_timeliness/blob/master/lib/validates_timeliness/extensions/date_time_select.rb#L12):

```
DEPRECATION WARNING: alias_method_chain is deprecated.
Please, use Module#prepend instead. From module, you can access the original method using super.
```

**Comment by adzap** [comment]
Thanks. Obviously haven't updated for Rails 5. Will likely bump gem to v5 and remove old Rails stuff. A clean break, including this change.

**Comment by dgilperez** [comment]
Hi! Any news? Is validates_timeliness Rails5 compatible?

**Comment by rposborne** [comment]
@adzap Anything we can do to help?

**Comment by hunzinker** [comment]
Hi! Any update on this issue?

Thanks!

**Comment by bjensen** [comment]
still not rails 5 compatible?

**Comment by svoop** [comment]
@adzap 

Given this issue and the last commit dating back to May 2017, I have to ask the question: Are you still maintaining these gems? It's totally okay if you have lost interest, happens to all of us sometimes, but I (and others I presume) have to decide where to go from here and whether to stick to "timeliness" and friends.

Also, there are 129 forks. If you ask the community, someone might step forward and continue your work by adopting the "timeliness" gems.

In any case, thanks a bunch for your work on these gems!

**Comment by adzap** [comment]
Hi all. I haven't given up, I just haven't gotten around to upgrading my own apps to Rails 5 which has always been to the trigger.  It all comes down to my availability which is has been terribly low I'm afraid. 

So my Rails 5 upgrade plans are within the 2 months. I have been reluctant to hand off this project off because I am still actively using and interested in maintaining and I really wanted to clean it up a lot and shed the cruft from supporting old AR versions.

Appreciate the interest and the frustration. 

Also currently there are no compelling PRs to do the upgrade. So it has not been a simple choice of merging a PR from the community which would accelerate all this. So if someone is interested in pulling together a PR of changes with specs we can review and discuss that would be excellent.

**Comment by svoop** [comment]
@adzap Great to know you're still on board! No worries though, I've been an "upgrade slut" myself as well. I'll have to check to what extent we're using timeliness and if we keep it, I'll surely PR the fixes our app needs to keep going.

**Comment by AquisTech** [comment]
@adzap I have faced this now as an exception as I am working on Rails 5.2.0.
I saw some recent commits of you regarding Rails 5 upgrades.
Please let me know if you are working on this particular `alias_method_change` issue. 
Else I can fix it and send you a PR. Thoughts?

**Comment by AquisTech** [comment]
@adzap  I have fixed this issue and raised PR #167 please review and merge it.

**Comment by adzap** [comment]
This specific issue is resolved after PR #167 but there is still more work to fix other issues before a Rails 5 release.
